File: 3206-find-common-elements-between-two-arrays/find-common-elements-between-two-arrays.cpp
class Solution {
public:
    vector<int> findIntersectionValues(vector<int>& nums1, vector<int>& nums2) 
    { 
       int ans1 = 0;
       for(int i=0;i<nums1.size();i++)
       {
            for(int j=0;j<nums2.size();j++)
            {
                if(nums1[i]==nums2[j])
                {
                    ans1++;
                    break;
                }
            }
       }
       int ans2 = 0;
       for(int i=0;i<nums2.size();i++)
       {
            for(int j=0;j<nums1.size();j++)
            {
                if(nums1[j]==nums2[i])
                {
                    ans2++;
                    break;
                }
            }
       }
       return {ans1,ans2};
    }

};

 

File: 3206-find-common-elements-between-two-arrays/find-common-elements-between-two-arrays.java
class Solution {
    public int[] findIntersectionValues(int[] nums1, int[] nums2) {

        int ans = 0;
        int ans1 = 0;

        
        for (int i = 0; i < nums1.length; i++) {
            for (int j = 0; j < nums2.length; j++) {
                if (nums1[i] == nums2[j]) {
                    ans++;
                    break;
                }
            }
        }

        
        for (int i = 0; i < nums2.length; i++) {
            for (int j = 0; j < nums1.length; j++) {
                if (nums2[i] == nums1[j]) {
                    ans1++;
                    break;
                }
            }
        }

        return new int[]{ans, ans1};
    }
}